packages/nextcloud/lib/src/api/theming.openapi.json

```diff
@@ -629,151 +629,6 @@
                 }
             }
         },
-        "/index.php/apps/theming/background/{type}": {
```

Comment:
Why did this operation get removed?

Reply:
There were some changes in the detection of some routes and this was previously detected as available, but in reality it is not. This isn't breaking as the endpoint would have never worked anyway.